en/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md

```diff
@@ -137,7 +137,7 @@ Unlike C++, in Ruby,...
   anyway). No casting either.
 * Iteration is done a bit differently. In Ruby, you don’t use a separate
   iterator object (like `vector<T>::const_iterator iter`) but instead
-  your objects may mixin the `Enumerator` module and just make a method
+  your objects may mixin the `Enumerable` module and just make a method
```

Comment:
Even so the sentence seems wrong, since `Enumerable` does not provide the `#each` method.